connector/exceptionsconnector/README.md

```diff
@@ -0,0 +1,87 @@
+# Exceptions Connector
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Distributions | [contrib] |
+
+[alpha]: https://github.com/open-telemetry/opentelemetry-collector#alpha
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+
+## Supported Pipeline Types
+
+| [Exporter Pipeline Type] | [Receiver Pipeline Type] | [Stability Level] |
+| ------------------------ | ------------------------ | ----------------- |
+| traces | metrics | [alpha] |
+| traces | logs | [alpha] |
+
+[Exporter Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#exporter-pipeline-type
+[Receiver Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#receiver-pipeline-type
+[Stability Level]: https://github.com/open-telemetry/opentelemetry-collector#stability-levels
+<!-- end autogenerated section -->
+
+## Overview
+
+Generate metrics and logs from record [application exceptions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-spans.md/) associated with spans.
+
+Each **metric** and **log** will have _at least_ the following dimensions:
+- Service name
+- Span kind
+- Status code
+
+With the provided default config, each **metric** and **log** will also have the following dimensions:
+- Exception message
+- Exception type
+
+Each log will additionally have the following attributes:
+- Exception stacktrace
+- HTTP attributes (if available), such as `http.method` and `http.host` among others.
```

Comment:
I forgot already, are you getting all HTTP attributes, or are you using a fixed list of attributes? It'd be good to be clear here. If it's all HTTP attributes, it would be good to state exactly which semantic convention version(s) you support.

And as a feature request: allow users to specify which semantic convention(s) they want to follow.